app/controllers/api/v1/keywords_controller.rb

```diff
@@ -11,6 +11,16 @@ def get_html_source
         end
       end
 
+      def get_html_page
```

Comment:
Regarding naming, it might be more expected to follow standard naming for controller methods, e.g., `show`.

```ruby
class KeywordsSourceController < BaseController
  def show
    #...
  end
end
```
Rails is based on conventions yielding to better code consistency but also making team collaboration easier. So in most cases, prefer to have numerous controllers instead of having few controllers with non-standard methods.

Reply:
I see, using the `KeywordsSourceController` controller would be a better fit for this API 😁